### ISSUE TITLE
Feature request: environment variables

### ISSUE THREAD

**Issue by lukasmrtvy** [vshn/rclone-exporter#4]
Would be great to have support for environment vars, at least for basic auth credentials.  In orchestrators like kubernes is practical to use env vars.
Thanks

**Comment by ccremer** [comment]
Hi. Thank you for the request.

Neither env vars, nor auth is yet supported.
In the meantime, what is stopping you from using K8s container arguments in the spec? e.g.
```yaml
args:
- --push.jobname
- my-job
- etc
```
The configuration effort is about the same as env vars.

**Comment by lukasmrtvy** [comment]
1. auth 
`--scrape.url http://admin:admin@rclone:5572` is working 

2. env vars
its much more deeper problem, trust me.

**Comment by ccremer** [comment]
Right, completely forgot about providing the credentials in the URL :/
Still, it's not ideal if the password pops up in any process lists which also lists arguments...

**Comment by ccremer** [comment]
Before we implement this, I would like to await the responses on the idea to integrate the exporter in the main rclone project in this discussion: https://github.com/rclone/rclone/issues/3858

**Comment by ccremer** [comment]
all options are now configurable with environment variables 🎉 

Please test it with the new release :)
